collections/transactionsCollection.js

```diff
@@ -91,3 +91,114 @@ TransactionsCollection.prototype._leagueFetch_callback = function(cb, e, data) {
 };
 
 // todo: http://fantasysports.yahooapis.com/fantasy/v2/league/{league_key}/transactions;types=waiver,pending_trade;team_key={team_key}
+
+TransactionsCollection.prototype.add_player = function(leagueKey, teamKey, playerKey, cb) {
+  var url = 'http://fantasysports.yahooapis.com/fantasy/v2/league/' + leagueKey + '/transactions?format=json';
+  var apiCallback = this._add_player_callback.bind(this, cb);
+  var xmlData = ' \
+    <fantasy_content> \
+      <transaction> \
+        <type>add</type> \
+        <player> \
+          <player_key>' + playerKey + '</player_key> \
+          <transaction_data> \
+            <type>add</type> \
+            <destination_team_key>' + teamKey + '</destination_team_key> \
+          </transaction_data> \
+        </player> \
+      </transaction> \
+    </fantasy_content>';
+
+  this
+    .yf
+    .api(
+      this.yf.POST,
+      url,
+      xmlData,
+      apiCallback
+    );
+};
+
+TransactionsCollection.prototype._add_player_callback = function(cb, e, data) {
+  if ( e ) return cb(e);
+
+  var meta = data.fantasy_content;
+  return cb(null, meta);
+};
+
+TransactionsCollection.prototype.drop_player = function(leagueKey, teamKey, playerKey, cb) {
+  var url = 'http://fantasysports.yahooapis.com/fantasy/v2/league/' + leagueKey + '/transactions?format=json';
+  var apiCallback = this._drop_player_callback.bind(this, cb);
+  var xmlData = ' \
+    <fantasy_content> \
+      <transaction> \
+        <type>drop</type> \
+        <player> \
+          <player_key>' + playerKey + '</player_key> \
+          <transaction_data> \
+            <type>drop</type> \
+            <source_team_key>' + teamKey + '</source_team_key> \
+          </transaction_data> \
+        </player> \
+      </transaction> \
+    </fantasy_content>';
+
+  this
+    .yf
+    .api(
+      this.yf.POST,
+      url,
+      xmlData,
+      apiCallback
+    );
+};
+
+TransactionsCollection.prototype._drop_player_callback = function(cb, e, data) {
+  if ( e ) return cb(e);
+
+  var meta = data.fantasy_content;
```

Comment:
I attached the response of each add/drop APIs. 😸  (I added/dropped my actual player in my team)
* [add_player_API.txt](https://github.com/whatadewitt/yfsapi/files/252271/add_player_API.txt)
* [adddrop_players_API.txt](https://github.com/whatadewitt/yfsapi/files/252272/adddrop_players_API.txt)
* [drop_player_API.txt](https://github.com/whatadewitt/yfsapi/files/252273/drop_player_API.txt)